src/test/integration/com/trzewik/information/producer/interfaces/rest/information/InformationControllerIT.groovy

```diff
@@ -32,7 +32,7 @@ class InformationControllerIT extends Specification implements ResponseVerifier,
         when:
             def response = getInformationRequest(information.id)
         then:
-            1 * informationService.get(information.id) >> information
+            1 * informationService.get(_) >> information
```

Comment:
for all tests! you should check that specific object was send to `informationService`. you can check it by:
```groovy
1 * informationService.get({GetInformationCommand command ->
assert commang.id == information.id
}) >> information
```

Reply:
here is example:
https://github.com/Czeffik/spring/blob/master/src/test/integration/com/trzewik/spring/interfaces/rest/game/GameControllerIT.groovy#L82